python/TestHarness/schedulers/Scheduler.py

```diff
@@ -226,27 +254,27 @@ def checkRaceConditions(self, dag_object):
             # Get a list of concurrent job containers
             concurrent_jobs = dag_clone.ind_nodes()
 
-            for job_container in concurrent_jobs:
-                tester = job_container.getTester()
+            for job in concurrent_jobs:
+                tester = job.getTester()
                 output_files = tester.getOutputFiles()
 
                 # check if we have colliding output files
                 if len(output_files_in_dir.intersection(set(output_files))):
 
                     # Fail this concurrent group of testers
                     for this_job in concurrent_jobs:
-                        tester = this_job.getTester()
-                        tester.setStatus('OUTFILE RACE CONDITION', tester.bucket_fail)
-                        failed_or_skipped_testers.add(tester)
+                        failed_tester = this_job.getTester()
+                        self.adjustJobStatus(this_job, failed_tester.bucket_fail, 'OUTFILE RACE CONDITION')
+                        failed_or_skipped_testers.add(failed_tester)
```

Comment:
how was THIS working before!?

Reply:
I'm not following, what problem do you see?

Reply:
Earlier in this nested for loop, i was already declaring tester. And then later, I clobber it with a different tester, and started setting statuses for it.